src/syngen/ml/vae/models/dataset.py

```diff
@@ -273,6 +275,28 @@ def _set_categorical_columns(self, df: pd.DataFrame, schema: Dict):
         self._define_categorical_columns(df)
         self._check_if_column_categorical(schema=schema)
 
+    def _set_long_text_columns(self, df: pd.DataFrame):
+        """
+        Set up the list of columns with long texts (> 200 symbols)
+        """
+        if self.schema.get("format", "") == "CSV":
+            data_subset = df.select_dtypes(include=[pd.StringDtype(), "object"])
+        else:
+            text_columns = [
+                col for col, data_type in self.schema.get("fields", {}).items()
+                if data_type == "string"
+            ]
+            data_subset = df[text_columns]
+        self.long_text_columns = set()
+        if not data_subset.empty:
+            data_subset = data_subset.loc[:, data_subset.apply(lambda x: (x.str.len() > 200).any())]
```

Comment:
Have you tested the case when text columns exist but none of them is a long text feature? I have a concern that this data_subset.apply(lambda x: (x.str.len() > 200).any()) will be an empty list an will raise the "Cannot index with multidimensional index" error

Reply:
I tested this case. It works okay)